### ISSUE TITLE
4080s 这个速度正常吗   1.67it/s

### ISSUE THREAD

**Issue by kennponn** [We-IOT/chatglm3_6b_finetune#16]
***** Running training *****
  Num examples = 70
  Num Epochs = 167
  Instantaneous batch size per device = 4
  Total train batch size (w. parallel, distributed & accumulation) = 4
  Gradient Accumulation steps = 1
  Total optimization steps = 3,000
  Number of trainable parameters = 1,949,696
{'loss': 3.5113, 'learning_rate': 4.9833333333333336e-05, 'epoch': 0.56}
{'loss': 3.3869, 'learning_rate': 4.966666666666667e-05, 'epoch': 1.11}
{'loss': 2.9629, 'learning_rate': 4.9500000000000004e-05, 'epoch': 1.67}
{'loss': 2.5854, 'learning_rate': 4.933333333333334e-05, 'epoch': 2.22}
{'loss': 2.3568, 'learning_rate': 4.9166666666666665e-05, 'epoch': 2.78}
{'loss': 1.9475, 'learning_rate': 4.9e-05, 'epoch': 3.33}
{'loss': 1.7979, 'learning_rate': 4.883333333333334e-05, 'epoch': 3.89}
{'loss': 1.5506, 'learning_rate': 4.866666666666667e-05, 'epoch': 4.44}
{'loss': 1.4642, 'learning_rate': 4.85e-05, 'epoch': 5.0}
  3%|██▍                                                                             | 90/3000 [06:43<29:06,  1.67it/s]

**Comment by We-IOT** [comment]
没有4080，性能上应该和3090差不多吧
